test/test_pipeline/components/setup/test_setup.py

```diff
@@ -441,7 +440,10 @@ def test_dummy_forward_backward_pass(self, task_type_input_output_shape):
         if task_type in constants.CLASSIFICATION_TASKS:
             dataset_properties["num_classes"] = output_shape[0]
 
-        cs = network_head_choice.get_hyperparameter_search_space(dataset_properties=dataset_properties)
+        cs = network_head_choice.get_hyperparameter_search_space(
```

Comment:
this issue should be fixed also, if you change the flag

Reply:
That is true, I will change the flag and revert the implementation.